### ISSUE TITLE
Wrong tile sequence in exported tilesets when using Packed sheet type

### ISSUE THREAD

**Issue by eefano** [aseprite/aseprite#4474]
Exported tileset in horizontal strip (CORRECT ORDER):

![right](https://github.com/aseprite/aseprite/assets/77832/034a4a04-75d1-48c5-b408-25dadbf74b3f)

Same tileset in packed (WRONG ORDER): 

![wrong](https://github.com/aseprite/aseprite/assets/77832/2470bef8-ed7a-40f5-8d64-6ba712b7c37b)

Packing size doesn't change the sequence.

The biggest problem is that the exported JSON is useless, because the tileset sequence doesn't match with the tile index in the tilemap.

Aseprite v1.3.6-dev on Arch Linux.

**Comment by dacap** [comment]
I think this could be solved with a new kind of sprite sheet format (Packed in Order?) that could be just the Horizontal Strip with a Fixed (packed) texture size. Which looks the most expected results (the Packed format is confusing because it's the only format that requires the JSON to recreate the original sprites).

**Comment by Gasparoken** [comment]
> I think this could be solved with a new kind of sprite sheet format (Packed in Order?) that could be just the Horizontal Strip with a Fixed (packed) texture size. Which looks the most expected results (the Packed format is confusing because it's the only format that requires the JSON to recreate the original sprites).

Yes! I agree. It's common for artists to confuse "Packed" with a kind of square-looking sprite sheet with its frames ordered by frame appearance. I think the most natural thing would be: If the JSON isn't requested in the checkbox in the Output section, it should result in "Packed in Order" (i.e., the result of `Sheet type: By Rows`, `Fixed # of Columns = ceil(sqrt(<number of frames>))`, force: uncheck Sprite and Cel trim and disabled, uncheck `Merge Duplicates` (Idk if `Merge Duplicates` = true would make sense in this case.)).
Other issues related to this:
- https://github.com/aseprite/aseprite/issues/2375
- https://github.com/aseprite/aseprite/issues/3608
- https://github.com/aseprite/aseprite/issues/5624
